htdocs/install/upgrade2.php

```diff
@@ -337,7 +338,7 @@
 			// Script for 2.9
 			$afterversionarray = explode('.', '2.8.9');
 			$beforeversionarray = explode('.', '2.9.9');
-			if (versioncompare($versiontoarray, $afterversionarray) >= 0 && versioncompare($versiontoarray, $beforeversionarray) <= 0) {
+			if ((versioncompare($versiontoarray, $afterversionarray) >= 0 && versioncompare($versiontoarray, $beforeversionarray) <= 0) || (versioncompare($versionfromarray, $afterversionarray) <= 0 && versioncompare($versiontoarray, $beforeversionarray) >= 0)) {
```

Comment:
Each function called may use structure of data that are migrated by the upgrade.php script.
So to migrate from x to x+2 we currently do 
upgrade to x+1
upgrade2 to x+1
upgrade to x+2
upgrade2 to x+2

With you change you do directly
upgrade2 to x+2
But when do the upgrade to x+2 is done ? Before ? It means we do
upgrade to x+2
upgrade2 to x+2 
?

Reply:
Yes we do upgrade x+1 _ upgrade x+2 _ upgrade x+n _ upgrade2 x+1 _upgrade2 x+2 _ upgrade2 x+n